packages/generator-reactserver/.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
Ima say this is out of scope; I created https://github.com/redfin/react-server/pull/2